authentication/two-factor.md

```diff
@@ -1,5 +1,5 @@
 # Two Factor Authentication
 
-Coming Soon!
+This is on the roadmap, but hasn't been started, yet.  Since `feathers-authentication@1.0.0+` runs on PassportJS, we can now build on top of Passport strategies like this one: https://www.npmjs.com/package/passport-totp. 
```

Comment:
Note to self. Mention `feathers-authentication-management`. It allows us to do 2 factor. Probably needs a guide in medium though....